etc/fetchers/fetch-wbw.js

```diff
@@ -1,7 +1,7 @@
 const cheerio = require("cheerio");
 const fs = require("fs");
 const path = require("path");
-const fetch = require("node-fetch");
+const { fetch } = require("cross-fetch");
```

Comment:
What's the benefit of this `cross-fetch` over `node-fetch`?

Reply:
@zainfathoni Works on both server-side and client-side.